config/cache.ts

```diff
@@ -0,0 +1,19 @@
+export const CACHE_DEFAULT_HEADERS =
+  'public, max-age=180, stale-if-error=1200, stale-while-revalidate=60';
+
+export const CACHE_ALLOWED_LIST_FILES_PATHS = [
```

Comment:
looks like it should be in this repo (https://github.com/lidofinance/warehouse/tree/main/packages/next/api-wrapper)

Reply:
This List for static files (eg manifest.json). In next-api-wrapper only api wrappers. To manage the cache-control with files, you need to use the next middleware. I may add it to the warehouse repo in the future

Reply:
you can provide something like `nextCacheAllowedList` variable to be exported for re-use (as default value)

Reply:
used @lidofinance/next-cache-files-middleware